src/components/CodeInput.tsx

```
@@ -0,0 +1,84 @@
+        <div
+            style={{
+                display: "flex",
+                gap: "15px",
+                marginBottom: "40px",
+                justifyContent: "center",
+            }}
+        >
+            {code.map((digit, index) => (
+                <TextField
+                    key={index}
+                    inputRef={(el) => (inputRefs.current[index] = el)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)